libraries/testAutomation/testAutomation/test_runner.py

```diff
@@ -201,14 +204,17 @@ def run(self, skip_delete: bool = False, keep_workflows: bool = False, developer
             reuse_clusters: bool = False, test_reports_dir: str = None, copy_logs: bool = False) -> None:
         self.k8s_service.create_namespace(self.k8s_service.namespace)
         if developer_mode:
-            workflow_templates_dir = (
-                "../../migrationConsole/"
-                "workflows/templates/"
+            integ_test_workflows_dir = (
+                "../../../migrationConsole/"
+                "lib/integ_test/testWorkflows/"
             )
-            self.k8s_service.run_command([
-                "kubectl", "apply", "-f", workflow_templates_dir, "-n", "ma"
-            ])
-            logger.info("Applied local workflow templates directory")
+            if os.path.isdir(integ_test_workflows_dir):
+                self.k8s_service.run_command([
+                    "kubectl", "apply", "-f", integ_test_workflows_dir, "-n", "ma"
+                ])
+                logger.info("Applied local integ test workflows directory")
+            else:
+                logger.info(f"Integ test workflows directory not found: {integ_test_workflows_dir}, skipping")
```

Comment:
This stumped me last round too - is this something that's typically used?  Is there a reason that we would need/want the test_runner, while in developer_mode, to do a (one-time?) apply on the resources?  This seems like a way to have inconsistencies creep in.